custom/acme/loan/job/src/main/resources/db/custom-changelog/0001_acme_loan_job.xml

```diff
@@ -42,4 +42,10 @@
             <column name="is_mismatched_job" valueBoolean="true"/>
         </insert>
     </changeSet>
+    <changeSet author="acme" id="2">
+        <update tableName="job">
+            <column name="short_name" value="ACM_NOOP"/>
+            <where>name='Acme Noop Job'</where>
+        </update>
+    </changeSet>
```

Comment:
@adamsaghy @marta-jankovics 

I am currently working on a custom module in another PR (#3994), and when I rebased the branch with the `develop` branch and ran the custom Docker image, I encountered the following error:

```sql
fineract-fineract-1  | Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'tenantDatabaseUpgradeService' defined in URL [jar:file:/app/libs/fineract-provider-0.1.0-SNAPSHOT-plain.jar!/org/apache/fineract/infrastructure/core/service/migration/TenantDatabaseUpgradeService.class]: liquibase.exception.CommandExecutionException: liquibase.exception.LiquibaseException: liquibase.exception.MigrationFailedException: Migration failed for changeset db/custom-changelog/0001_acme_loan_job.xml::1::acme:
fineract-fineract-1  |      Reason: liquibase.exception.DatabaseException: ERROR: null value in column "short_name" of relation "job" violates not-null constraint
fineract-fineract-1  |   Detail: Failing row contains (40, Acme Noop Job, Acme Noop Job, 0 1 0 1/1 * ? *, 2024-08-29 15:30:47.958143, 5, null, null, null, Acme Noop Job _ DEFAULT, null, f, f, t, 0, f, 1, t, null). [Failed SQL: (0) INSERT INTO public.job (name, display_name, cron_expression, create_time, task_priority, group_name, previous_run_start_time, job_key, initializing_errorlog, is_active, currently_running, updates_allowed, scheduler_group, is_misfired, node_id, is_mismatched_job) VALUES ('Acme Noop Job', 'Acme Noop Job', '0 1 0 1/1 * ? *', NOW(), 5, NULL, NULL, 'Acme Noop Job _ DEFAULT', NULL, FALSE, FALSE, TRUE, 0, FALSE, 1, TRUE)]
```

Could you please explain why this occurred and how I can resolve it?

> [!NOTE]
> I am running [`docker-compose-api-v3.yml`](https://github.com/Zeyad2003/fineract/blob/c63762763f40cfb8390bdbf53a9eb6c425f63e0c/docker-compose-api-v3.yml) which uses **Postgres** image

Reply:
I guess the problem might be the order of execution of the liquibase scripts. @marta-jankovics What do you think?

Reply:
0001_acme_loan_job.xml
should run before 
0146_add_final_constraints.xm
make sure that in
db/changelog/db.changelog-master.xml
your module changeling-master.xml is listed before
tenant/final-changelog-tenant.xml

Reply:
The problem is that scripts marked with custom_changelog run after the normal run was finished. This is not correct. I'll fix this with a a small PR today.